samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -862,6 +1045,20 @@ int main(void)
 {
 	int err;
 
+#if defined(CONFIG_USB_DEVICE_AUDIO)
+	/* Use this to turn on 128 MHz clock for cpu_app, this improves the  */
+	err = nrfx_clock_divider_set(NRF_CLOCK_DOMAIN_HFCLK, NRF_CLOCK_HFCLK_DIV_1);
+
+	err -= NRFX_ERROR_BASE_NUM;
+	if (err) {
+		return err;
+	}
+
+	nrfx_clock_hfclk_start();
+	while (!nrfx_clock_hfclk_is_running()) {
+	}
+#endif /* defined(CONFIG_USB_DEVICE_AUDIO) */
```

Comment:
Is this necessary?

Reply:
It's a very good question, and please feel free to involve anyone else from Nordic understanding this better.

I see (or hear) much better result when I do include this, esp. on 24Khz. 16Khz sounds good with or without, but 24Khz is really struggling, like you are listening to radio in your car and entering a tunnel without it.
There are still a few "sparks" left even with this, but its a huge improvement.

We did see similar results on source previously as well, but IIRC @larsgk said that that solved itself once the threads were offloaded. But it doesn't seem to have helped here.

Reply:
Have you tried running the thread analyzer to see if we are at 100% CPU usage? From what I've seen before, decoding a single 16KHz stream only uses something like 17% usage without this overclocking, so I'm surprised you are seeing it. 

If you are hearing issue with the audio and the CPU is not at 100%, then the issue may lie elsewhere and should probably be figured out rather than just pumping up the CPU